doc/classes/EditorPaths.xml

```diff
@@ -1,35 +1,65 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorPaths" inherits="Object" version="4.0">
 	<brief_description>
+		Editor-only singleton that returns paths to various OS-specific data folders and files.
 	</brief_description>
 	<description>
+		This editor-only singleton returns OS-specific paths to various data folders and files. It can be used in editor plugins to ensure files are saved in the correct location on each operating system.
+		[b]Note:[/b] This singleton is not accessible in exported projects. Attempting to access it in an exported project will result in a script error as the singleton won't be declared. To prevent script errors in exported projects, use [method Engine.has_singleton] to check whether the singleton is avilable before using it.
```

Comment:
```suggestion
		[b]Note:[/b] This singleton is not accessible in exported projects. Attempting to access it in an exported project will result in a script error as the singleton won't be declared. To prevent script errors in exported projects, use [method Engine.has_singleton] to check whether the singleton is available before using it.
```